workflow/util/util.go

```diff
@@ -193,6 +194,21 @@ func CreateServerDryRun(wf *wfv1.Workflow, wfClientset wfclientset.Interface) (*
 	return wf, err
 }
 
+func PopulateSubmitOpts(command *cobra.Command, submitOpts *wfv1.SubmitOpts, includeDryRun bool) {
```

Comment:
boo to util.go - can you move under `cmd/argo` directory please

Reply:
I move this to util.go because:

1. Adding this on `cmd/argo` would create an import cycle when this is imported in `cmd/argo/cron`. I could create `cmd/util` if necessary
1. `ApplySubmitOpts` is right below this